src/components/search/character-check.js

```diff
@@ -40,27 +42,29 @@ export default class CharCheck {
     let message;
     if (remaining === 1) {
       message = this.singularMessage;
-    } else if (remaining === -1) {
+    } else if (remaining > 1) {
+      message = this.pluralMessage;
+    } else if (remaining === -1 && !this.checkMinOnly) {
       message = this.charLimitSingularMessage;
       remaining = Math.abs(remaining);
-    } else if (remaining < -1) {
+    } else if (remaining < -1 && !this.checkMinOnly) {
       message = this.charLimitPluralMessage;
       remaining = Math.abs(remaining);
-    } else {
-      message = this.pluralMessage;
     }
     this.setShowMessage(remaining);
     this.setButtonState(remaining);
     this.checkElement.innerText = message.replace('{x}', remaining);
```

Comment:
@matthewandrewpalmer In this new variation this line is creating an error in the console. It's because `message` isn't defined after the limit is reached. You'll probably have wrap this line in an `if` to check if `message` exists. `if (message) {}` etc